cypress/integration/ambianic-tests/navbar.spec.js

```diff
@@ -1,23 +1,26 @@
 /// <reference types="cypress" />
 
+export const checkViewPort = (cy, device) => {
+  cy.viewport(device)
+  expect(cy.get("[data-cy=login]")).to.exist
+}
+
 context('Check Navbar Items', () => {
   before(() => {
     cy.visit('http://localhost:8080')
     cy.get('[data-cy=timeline]').click()
   })
 
+  it('Should have a search bar', () => {
+    cy.get('[data-cy=container').find('#searchbar')
```

Comment:
@vickywane what post condition do we expect on this test?

Reply:
We should probably disable the search bar until its ready to provide some value.

Reply:
> what post condition do we expect on this test?

I didnt write that test.

> We should probably disable the search bar until its ready to provide some value.

Okay. We can do this.